README.md

```diff
@@ -1,40 +1,41 @@
 # Welcome to the lab on enabling GitHub Pages!
 
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
+This lab is an exercise to check your knowledge on enabling GitHub Pages. It is automatically graded via a workflow once you have completed the instructions.
```

Comment:
I love this TLDR, and it addresses many of the early feedback I got that the context of the exercise wasn't easily accessible before.

Reply:
Yeah I thought it would be good to set the expectation for the learner right away!